src/views/user/LogEntries/LogEntry.js

```diff
@@ -97,11 +98,10 @@ function LogEntry({ className }) {
     }
   };
 
-  const date = new Date().toLocaleTimeString('en-US', {
-    hour12: false,
-    hour: 'numeric',
-    minute: 'numeric'
-  });
+  const amount = logEntryKind.attributes.amount
+    ? convertAmount(logEntryKind.attributes.amount)
+    : convertAmount(logEntryKind.attributes.duration);
```

Comment:
I'm not sure if the same conversion formula would work for a nap's duration. Maybe we can discuss tomorrow.